framework/docker/ibc/relayer/hermes_types.go

```diff
@@ -0,0 +1,36 @@
+package relayer
```

Comment:
these types map to the stdout of various hermes cli commands, to more easily parse them and return concrete types.